docs/painless/painless-contexts/painless-reindex-context.asciidoc

```diff
@@ -0,0 +1,68 @@
+[[painless-reindex-context]]
+=== Reindex Context
+
+Use a Painless script in a {es_version}/docs-reindex.html[reindex] operation to
+add, modify, or delete fields within each document in an original index as its
+re-indexed into a target index.
+
+*Variables*
+
+`params` (`Map`, read-only)::
+        User-defined parameters passed in as part of the query.
+
+`ctx['_op']` (`String`)::
+        The name of the operation.
+
+{es_version}/mapping-routing-field.html[`ctx['_routing']`] (`String`)::
+        The value used to select a shard for document storage.
+
+{es_version}/mapping-index-field.html[`ctx['_index']`] (`String`)::
+        The name of the index.
+
+{es_version}/mapping-type-field.html[`ctx['_type']`] (`String`)::
+        The type of document within an index.
+
+{es_version}/mapping-id-field.html[`ctx['_id']`] (`int`, read-only)::
+        The unique document id.
+
+`ctx['_version']` (`int`)::
+        The current version of the document.
+
+{es_version}/mapping-source-field.html[`ctx['_source']`] (`Map`)::
+        Contains extracted JSON in a `Map` and `List` structure for the fields
+        existing in a stored document.
+
+*Side Effects*
+
+`ctx['_op']`::
+        Use the default of `'index'` to update a document. Set to `'none'` to
+        specify no operation or `'delete'` to delete the current document from
+        the index.
```

Comment:
The single quotes aren't necessary, the monospace style is sufficient.

Reply:
Fixed.